packages/grafana-ui/src/components/Logs/LogRow.tsx

```diff
@@ -59,58 +69,72 @@ class UnThemedLogRow extends PureComponent<Props, State> {
   ) {
     const {
       getRows,
+      onClickFilterLabel,
+      onClickFilterOutLabel,
       highlighterExpressions,
-      onClickLabel,
       row,
       showDuplicates,
-      showLabels,
       timeZone,
       showTime,
       theme,
     } = this.props;
-    const { showContext } = this.state;
+    const { showDetails, showContext } = this.state;
     const style = getLogRowStyles(theme, row.logLevel);
     const showUtc = timeZone === 'utc';
-
     return (
-      <div className={cx([style.logsRow])}>
-        {showDuplicates && (
-          <div className={cx([style.logsRowDuplicates])}>
-            {row.duplicates && row.duplicates > 0 ? `${row.duplicates + 1}x` : null}
-          </div>
-        )}
-        <div className={cx([style.logsRowLevel])} />
-        {showTime && showUtc && (
-          <div className={cx([style.logsRowLocalTime])} title={`Local: ${row.timeLocal} (${row.timeFromNow})`}>
-            {row.timeUtc}
+      <>
+        <div className={cx([style.logsRow])}>
+          {showDuplicates && (
+            <div className={cx([style.logsRowDuplicates])}>
+              {row.duplicates && row.duplicates > 0 ? `${row.duplicates + 1}x` : null}
+            </div>
+          )}
+          <div className={cx([style.logsRowLevel])} />
+          <div onClick={this.toggleDetails} className={cx([style.logsRowToggleDetails])}>
+            <i className={showDetails ? 'fa fa-chevron-up' : 'fa fa-chevron-down'} />
           </div>
-        )}
-        {showTime && !showUtc && (
-          <div className={cx([style.logsRowLocalTime])} title={`${row.timeUtc} (${row.timeFromNow})`}>
-            {row.timeLocal}
-          </div>
-        )}
-        {showLabels && (
-          <div className={cx([style.logsRowLabels])}>
-            <LogLabels
-              getRows={getRows}
-              labels={row.uniqueLabels ? row.uniqueLabels : {}}
-              onClickLabel={onClickLabel}
-            />
+          <div>
+            <div>
+              {showTime && showUtc && (
+                <div className={cx([style.logsRowLocalTime])} title={`Local: ${row.timeLocal} (${row.timeFromNow})`}>
+                  {row.timeUtc}
+                </div>
+              )}
+
+              {showTime && !showUtc && (
+                <div className={cx([style.logsRowLocalTime])} title={`${row.timeUtc} (${row.timeFromNow})`}>
+                  {row.timeLocal}
+                </div>
+              )}
+              <div className={cx([style.logsRowCell])}>
```

Comment:
Just a small tip that your using the cx utility function everywhere:
1)  first this function is not needed as it is only for composing multiple styles together (and for composing legacy classnames plus emotion styles)
2) don't think it requires the array syntax either 
3) So int the above case youy can just write className={style.logRowCell}

Reply:
Yes, you are right. 🙂Updated! Thanks for the feedback.